AEXS/aex-141.md

```diff
@@ -282,7 +296,7 @@ Approval(address, address, int, string)
 | token_id | int |
 | enabled | string |
 
-## *ApprovalForAll*
+### *ApprovalForAll*
 
 This event MUST be triggered and emitted upon a change of operator status, including revocation of approval.
```

Comment:
this description doesn't include the "All" part

Reply:
fixed: https://github.com/aeternity/AEXs/commit/7a96c18d882c83541fb1701b1a37f3777602d76a